downstream/modules/automation-hub/con-offline-token-active.adoc

```diff
@@ -0,0 +1,16 @@
+
+[id="con-offline-token-active"]
+
+= Keeping your offline token active
+
+Keeping an offline token active is useful in the event that an application needs to perform action on behalf of the user, even when the user is offline. For example, a routine data backup.
```

Comment:
Suggest rewording to simplify:
Keeping an offline token active is useful when an application needs to perform....